opentech/apply/funds/templates/funds/base_submissions_table.html

```diff
@@ -11,11 +11,11 @@
         <div class="wrapper wrapper--table-actions">
             <button class="button button--filters button--contains-icons js-toggle-filters">Filters</button>
 
-            <form method="get" role="search" class="form form--search-desktop js-search-form">
+            <form method="get" role="search" class="form form--search js-search-form">
                 <button class="button button--search" type="submit" aria-label="Search">
                     <svg class="icon icon--magnifying-glass icon--search"><use xlink:href="#magnifying-glass"></use></svg>
                 </button>
-                <input class="input input--search js-search-input" type="text" placeholder="Search submissions" name="query"{% if search_term %} value="{{ search_term }}"{% endif %} aria-label="Search input">
+                <input class="input input--secondary input--search js-search-input" type="text" placeholder="Search submissions" name="query"{% if search_term %} value="{{ search_term }}"{% endif %} aria-label="Search input">
```

Comment:
Is the `input--secondary` class necessary? I can't see the `--secondary` variant in `_input.scss`

Reply:
we're using it as part of the `:not()` selector in `__form.scss` to basically reset the styles